services/kubernetesbuilddeploy/src/index.js

```diff
@@ -134,6 +134,9 @@ const messageConsumer = async msg => {
       "metadata": {
           "creationTimestamp": null,
           "name": buildName,
+          "labels": {
+            "lagoon.job.type": "build",
```

Comment:
can we use the label `lagoon.sh/jobType` ? (see https://github.com/amazeeio/lagoon/pull/1835)

Reply:
Done!